frontend/src/hooks/hooks.js

```diff
@@ -19,3 +23,32 @@ export const useGetPreserveParamsHref = () => {
             : `?${oldSearchParams.toString()}`
     return (path) => `${path}${newSearchParams}`
 }
+
+export const useGetCurrentTicket = () => {
+    const location = useLocation()
+    const allTickets = useRecoilValue(allTicketsState)
+    const getCurrentTicket = useCallback(() => {
+        const splitPath = location.pathname.split('/')
+        if (splitPath.length !== 3) return {}
+        let ticketType = splitPath[1]
```

Comment:
```suggestion
        let ticketType = splitPath[1].toUpperCase()
```
The app crashes when visiting something like `http://localhost:3000/upr/1` because the route still matches but it fails to find UPR-1 in alltickets because the keys are case sensitive